src/mono/browser/Makefile

```diff
@@ -167,4 +167,4 @@ build-app-host:
 	$(DOTNET) build $(TOP)/src/mono/wasm/host $(_MSBUILD_WASM_BUILD_ARGS) $(MSBUILD_ARGS)
 
 patch-deterministic:
-	cd emsdk/upstream/emscripten/ && patch -p1 < ../../../runtime/deterministic.diff
+	cd emsdk/emscripten/ && patch -p1 < ../../../runtime/deterministic.diff
```

Comment:
Please add a comment explaining why this is here, while you're touching it. The diff isn't self-explanatory

Reply:
yeah, the gist is that if you clone emsdk and install, it'll pull a bunch of things and put them in the 'upstream' directory. Our source built packages don't have this directory structure.

Reply:
Sorry, I mean that deterministic.diff isn't self-explanatory

Reply:
it predates this PR. it was added by https://github.com/dotnet/runtime/pull/75532, not sure what was the context there. @lewing do you remember?

Reply:
This was something Zoltan used to make the each run deterministic so that he could do direct comparisons, it is only for development/testing